include/sys/fs/zfs.h

```diff
@@ -906,10 +909,14 @@ typedef struct zpool_load_policy {
 #define	ZPOOL_CONFIG_DRAID_NSPARES	"draid_nspares"
 #define	ZPOOL_CONFIG_DRAID_NGROUPS	"draid_ngroups"
 
+/* ANYRAID configuration */
+#define	ZPOOL_CONFIG_ANYRAID_PARITY_TYPE	"parity_type"
```

Comment:
nit: should the string be "anyraid_parity_type", just so it's clear if someone is printing the config?